### PR TITLE
Format DateTime now to second

### PR DESCRIPTION
**Pull request by dvdn**
# Description

This PR fixes difference in Datetime rendered value between a 
- nominal case : disruption/impact values passing via RMQ to Navitia (it doesn't round datetime second value)
- Navitia reinit case : disruption/impact values directly read in Chaos DB (it rounds datetime to closest second)

## Issue (optional)

Issue link: BOT-569

## Pull Request type (optional)

bug fix

## How to test

- Submit a disruption to Navitia and compare  'created_at' recorded in DB Chaos and 'updated_at' in Navitia
- Stop and restart Navitia to reload disruptions, and compare  'created_at' recorded in DB Chaos and 'updated_at' in Navitia

It should always be the same

### CODE REVIEW

File: chaos/utils.py [hunk 1]
```diff
@@ -179,12 +179,19 @@ def get_utc_datetime_by_zone(value, time_zone):
 def get_current_time():
```
Comment (Netmisa) [inline chaos/utils.py]: I think you can do `get_current_time(format)` call with format argument.
With that you can remove `get_current_time_format_second` function

**Comment by Netmisa** [comment]
You can use `datetime.utcnow().replace(microsecond=0)` in `get_current_time()` function

**Comment by dvdn** [comment]
Yo bro, retest this please !

**Comment by dvdn** [comment]
Yo bro, retest this please !

**Comment by dvdn** [comment]
Yo bro, retest this please !

**Pull request merged**